plugins/apollo-explorer/README.md

```diff
@@ -69,3 +69,41 @@ That's it! You should now see an `Apollo Explorer` item in your sidebar, and if
 Once you authenticate, your graph is ready to use 🚀
 
 ![Logged In](./docs/img/logged-in.png)
+
+### Authentication Tokens for Apollo Studio
+
+If you need to utilize an ApiRef to supply a token to Apollo, you may do so using an ApiHolder.
+
+In `packages/app/src/App.tsx` perform the following modifications from above. The import `ssoAuthApiRef` is used as an example
+
+```typescript
+import { ApolloExplorerPage, EndpointProps } from '@backstage/plugin-apollo-explorer';
+import { ssoAuthApiRef } from '@backstage/devkit';
+import { ApiHolder } from '@backstage/core-plugin-api';
+
+
+async function apolloPluginEndpointsCallback(options: { apiHolder: ApiHolder }): Promise<EndpointProps[]> {
+  const sso = options.apiHolder.get<any>(ssoAuthApiRef)
```

Comment:
```suggestion
  const sso = options.apiHolder.get(ssoAuthApiRef)
```

I expect this to work as type should be inferred from the API ref and the type should be correct